docs/getting-started/integrations/angular.md

```diff
@@ -690,6 +690,19 @@ export class AppComponent {
 
 For advanced usage see the {@link getting-started/setup/ui-language Setting the UI language} guide.
 
+<info-box warning>
+There is a known issue related to the localization in Angular 17. Read more in the [known issues section](#known-issues) below.
+</info-box>
```

Comment:
I was wondering how to communicate this. I was thinking about three options:

1. Just a paragraph (It was easily missable).
2. Warning box at the start of the localization section (It was "scary").
3. Finally, I stick to the warning box at the end of the section.